Quickstart/azure-search-quickstart/azure-search-quickstart/Program.cs

```diff
@@ -0,0 +1,267 @@
+        static void Main(string[] args)
+        {
+            IConfigurationBuilder builder = new ConfigurationBuilder().AddJsonFile("appsettings.json");
+            IConfigurationRoot configuration = builder.Build();
+
+            SearchServiceClient serviceClient = CreateSearchServiceClient(configuration);
+
+            string indexName = configuration["SearchIndexName"];
+
+            Console.WriteLine("{0}", "Deleting index...\n");
+            DeleteIndexIfExists(indexName, serviceClient);
+
+            Console.WriteLine("{0}", "Creating index...\n");
+            CreateIndex(indexName, serviceClient);
+
+            // Uncomment next 3 lines in "2 - Load documents"
+            // ISearchIndexClient indexClient = serviceClient.Indexes.GetClient(indexName);
+            // Console.WriteLine("{0}", "Uploading documents...\n");
+            // UploadDocuments(indexClient);
+
+            // Uncomment next 2 lines in "3 - Search an index"
+            // Console.WriteLine("{0}", "Searching index...\n");
+            // RunQueries(indexClient);
+
+            Console.WriteLine("{0}", "Complete.  Press any key to end application...\n");
+            Console.ReadKey();
+        }
+
+        // Create the search service client
+        private static SearchServiceClient CreateSearchServiceClient(IConfigurationRoot configuration)
+        {
+            string searchServiceName = configuration["SearchServiceName"];
+            string adminApiKey = configuration["SearchServiceAdminApiKey"];
+
+            SearchServiceClient serviceClient = new SearchServiceClient(searchServiceName, new SearchCredentials(adminApiKey));
+            return serviceClient;
+        }
+
+        // Delete an existing index to reuse its name
+        private static void DeleteIndexIfExists(string indexName, SearchServiceClient serviceClient)
+        {
+            if (serviceClient.Indexes.Exists(indexName))
+            {
+                serviceClient.Indexes.Delete(indexName);
+            }
+        }
+
+        // Create an index with a single call to "Indexes.Create"
+        // This method takes as a parameter an Index object that defines an Azure Search index
+        // 
+        // Set the Fields property of the Index object to an array of Field objects.
+        // The field array is defined in the Hotels class, which subsumes the Address class.
+        private static void CreateIndex(string indexName, SearchServiceClient serviceClient)
+        {
+            var definition = new Index()
+            {
+                Name = indexName,
+                Fields = FieldBuilder.BuildForType<Hotel>()
+            };
+
+            serviceClient.Indexes.Create(definition);
+        }
+
+        // Upload documents as a batch
+        private static void UploadDocuments(ISearchIndexClient indexClient)
+        {
+            var actions = new IndexAction<Hotel>[]
+            {
+            IndexAction.Upload(
```

Comment:
nit: Indentation is wacky here. The closing brace of the array literal should align with the opening brace, and the IndexAction.Upload calls should be indented four spaces from the opening brace.

Reply:
Fixed